.ado/pipelines/templates/stages-full-release.yaml

```diff
@@ -456,6 +479,34 @@ stages:
 
               - template: steps-set-pipeline-variables.yaml # load set-pipeline-variables function
 
+              # Delete abandoned Load Test resource (if still present)
+              # There might be edge cases when load testing artifacts remain and need to be cleaned up
+              - task: AzureCLI@2
+                displayName: 'Cleanup abandoned load testing artifacts (if needed)'
+                retryCountOnTaskFailure: 1
+                inputs:
+                  azureSubscription: $(azureServiceConnection)
+                  scriptType: pscore
+                  scriptLocation: inlineScript
+                  inlineScript: |
+                    $maltRg="$(prefix)$(customSuffix)-azloadtest-rg"
```

Comment:
@heoleri why dont we just use the same RG name for both, MALT and Locust? than this is just a single deletion logic.

Reply:
mh.. yes.. that makes sense

Reply:
They're now both using `<prefix>-loadtest-rg`